tools/run_gpu_tests.sh

```diff
@@ -9,4 +9,4 @@ docker build \
        --build-arg TF_VERSION=2.2.0 \
        --build-arg PY_VERSION=3.5 \
        -t tfa_gpu_tests ./
-docker run --rm -t -v cache_bazel:/root/.cache/bazel --gpus=all tfa_gpu_tests
+docker run --rm -t --gpus=all tfa_gpu_tests
```

Comment:
It is working so I don't know if anybody remember why the bazel cache volume was required here but we could revert this change.